appveyor.yml

```diff
@@ -19,6 +23,8 @@ deploy: off
 
 on_finish:
     - ps: |
+        # Only publish one artifact
+        if(-not $IsWindows) { return }
```

Comment:
Not sure which version of PS appveyor image has, let's make it more explicit in case `$IsWindows -eq $null`.
I.e. `if ($IsWindows -eq $False)`

Reply:
Doh, good catch.